### PR TITLE
Make config prettier compatible

### PR DESCRIPTION
**Pull request by GuyHadas**
This removes all the style linting rules that conflicted with prettier's opinionated format.

### CODE REVIEW

**Review by ariwbolton — COMMENTED** [review]
Am trusting that you've tested this because I definitely haven't! Looks in line w/ what I'd expect from this type of PR, though -- nice working methodically going through and figuring out exactly what needs to change 👍 

Have one major question: if this change is meant to ensure that this configuration is compatible w/ prettier, it seems like we should have tests against that requirement -- should we add some?

Side note: Would also recommend updating the README to note that it is intended to be used w/ Prettier because to other consumers (this repo is public!) might not know what caused the extreme change. Would also definitely recommend doing a major version bump as part of that, because of how big a change this is.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ Then, add an `.eslintrc.json` file with the following:
 
 Then, try running like `npx eslint .`
 
+# Philosophy
+
+Originally, Scoop's eslint configuration was designed to support both Scoop's style guidelines as well as catch syntactical errors. After some evolution, [Prettier](https://prettier.io/) was adopted by Scoop's Frontend
+team, and then eventually expanded to our backend code as well. Our goal is to have our eslint catch all of our syntactical issues, while letting Prettier decide our formatting decisions. No eslint style rule should ever
```
Comment (seanking11) [inline README.md:22]: Minor: I find `syntactical` and `formatting` to be pretty synonymous. In order to make it a bit clearer about the intent, I suggest we adopt the language used by Prettier and include an example to drive the point home that eslint is meant to catch bugs and prettier is meant to format.

```suggestion
team, and then eventually expanded to our backend code as well. Our goal is to have our eslint catch all of our code-quality issues (`no-unused-vars`), while letting Prettier decide our formatting decisions (`max-len`). No eslint style rule should ever
```

**Review by seanking11 — APPROVED** [review]
Thanks for taking this on -- I'm curious, how did you go about figuring out the rules that conflict with prettier's opinions? On prettier's [docs](https://prettier.io/docs/en/integrating-with-linters.html), they mention an [eslint config](https://github.com/prettier/eslint-config-prettier) they created that disables any eslint rules that might conflict with prettier. Not sure if you've seen already but could be useful to look at.

Going to approve as my comment is minor, though I do suggest the addition for clarity

**Comment by GuyHadas** [comment]
@seanking11 I just manually added prettier to both the api and cosmos and saw which rules of ours it failed. Figured our sample size was definitely large enough to catch any issues.

**Pull request merged**